tests/examples/awful/widget/layoutlist/bar.lua

```diff
@@ -0,0 +1,62 @@
+local awful = { --DOC_HIDE --DOC_NO_USAGE
```

Comment:
Could you please also do a test showing the "default behaviour" of the layoutlist, i.e. only the minimum required arguments are given (`screen = 1`, I think). That way, (a) I'm more convinced that the widget actually works by default and (b) people get an idea what the widget actually looks like by default.

Reply:
done